README.md

```diff
@@ -166,7 +167,12 @@ For each `Animal`:
 
 #### Constraints and Rules
 - `@optional` can only be applied to vertex fields, except the root vertex field.
+- It is allowed to expand vertex fields within an `@optional` scope.
+  However, doing so is currently associated with a performance penalty in `MATCH`.
+  For more detail, see: [Traversing within `@optional`](#traversing-within-optional-optional-).
 - May not exist at the same vertex field as `@recurse`, `@fold`, or `@output_source`.
+- `@optional`, `@output_source` and `@fold` may not be used anywhere within a scope
```

Comment:
I think there's some overlap with previous documentation in here in the line before (or maybe it's the phrasing?)